lab-shelly/test/signup-test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
```

Comment:
Note you'll probably have to mock a gallery and then testing the update function.